modin/pandas/test/dataframe/test_indexing.py

```diff
@@ -562,6 +562,46 @@ def test_loc_multi_index():
     df_equals(modin_df.loc[modin_df.index[:7]], pandas_df.loc[pandas_df.index[:7]])
 
 
+def test_loc_multi_index_with_tuples():
+    arrays = [
+        ["bar", "bar", "baz", "baz", "foo", "foo", "qux", "qux"],
+        ["one", "two", "one", "two", "one", "two", "one", "two"],
+    ]
```

Comment:
Since we only need the `("bar", "two")` tuple, let's git rid of some of the extra tuples.

Reply:
I want to keep a few of them for some future tests potentially. I'll remove some of them.